app/controllers/proceeding_types/searches_controller.rb

```diff
@@ -0,0 +1,79 @@
+module ProceedingTypes
+  class SearchesController < ApplicationController
+    resource_description do
+      short 'Search proceeding types'
+      formats(%w[json])
+      description <<-END_OF_TEXT
+      == Description
+      This endpoint accepts a json field containing a single value, `search_term`
+        {
+          search_term: 'occup'
+        }
+      and it will return a success marker and a data array of proceeding_types in json format, e.g.
+        {
+          success: true,
+          data: [
+            {
+              meaning: 'Occupation order',
+              ccms_code: 'DA005',
+              description: 'to be represented on an application for an occupation order.',
+              ccms_category_law: 'Family',
+              ccms_matter: 'Domestic abuse'
+            }
+          ]
+        }
+
+      Optionally you can send an excluded_terms value, this should be a comma separated string or codes to exclude, e.g.
+        {
+          "search_term": "Family",
+          "excluded_terms": "DA001,DA0002"
+        }
+      This will return proceeding terms that match Family but exclude any proceeding types with matching codes
+      END_OF_TEXT
+    end
+
+    api :POST, 'proceeding_types/search', 'Create a request to retrieve a list of proceeding types that match the search type'
+    param :search_term, String, required: true, desc: 'Search for proceeding types matching the `search_text`'
```

Comment:
line 8 and 28 refer to `search_term` but this line refers to `search_text`, `search_term` seems more appropriate

Reply:
It _was_ search_text originally but I changed the value and forgot the docs!